apps/flame-defi/app/features/evm-wallet/services/non-fungible-position-manager-service/index.ts

```diff
@@ -282,6 +296,98 @@ export class NonfungiblePositionManagerService extends GenericContractService {
     );
   }
 
+  /**
+   * Collect fees and tokens from a position with support for wrapped native tokens and multicall operations.
+   *
+   * This method handles both wrapped native token collection and non-native token collection scenarios.
+   */
+  async collectFeesV2({
+    chain,
+    tokenId,
+    token0,
+    token1,
+    position,
+    recipient,
+    options,
+    ...params
```

Comment:
Since there's only prop in these "rest params", I think this is functionally equivalent and removes the need for line 314
```suggestion
    calls = [],
```